### ISSUE TITLE
Can't build theia-endpoint-runtime because of an API breakage on Theia 0.10.x-next

### ISSUE THREAD

**Issue by monaka** [eclipse-che/che#14273]
### Describe the bug

I couldn't build Che-theia. I can't show how to reproduce because my build environment is customized.
But the root cause is obvious.
`getPluginDependencies` is not added to `PluginDeployerHandlerImpl` even though it was added to `PluginDeployerHandler` (ref: https://github.com/theia-ide/theia/commit/2d4c8457c09cc6ca37fc5480d14eb080a74ed85a#diff-2a5766562161792548814f3e490b89ffR613-R614 )

### Che-Theia version
 - [x] other: please specify (master branch with some patches.)

### Expected behavior

Builds finish with no error.

### Additional context

```
packages/theia-remote/src/node/plugin-remote-init.ts(290,7): error TS2420: Class 'PluginDeployerHandlerImpl' incorrectly implements interface 'PluginDeployerHandler'.
  Property 'getPluginDependencies' is missing in type 'PluginDeployerHandlerImpl'.
```

My build log is https://dev.azure.com/pizzafactory/camino/_build/results?buildId=1229

**Comment by monaka** [comment]
This might be fixed by eclipse/che-theia#406 ?

**Comment by rhopp** [comment]
@evidolob I've set P1 for this issue. Do you think your PR will resolve this one?

**Comment by monaka** [comment]
It looks to be fixed by eclipse/che-theia#409. I close it.
